src/containers/UserEdit.jsx

```diff
@@ -168,6 +169,7 @@ class UserEdit extends React.Component {
                 name="lastName"
                 {...dataTest("lastName")}
               />
+              <Form.Field label="Alias" name="alias" {...dataTest("alias")} />
```

Comment:
Interested in thoughts from others -- I feel like "Alias" doesn't have much context especially when the user is just creating their account.  Maybe label (or if there's a hint/description property) could be "Alias for texting" -- and make it clear someone can leave it blank?

Reply:
In my mockup I believe I used "Texting Alias"

Reply:
Is everybody fine if I change it to `Texting Alias`?

Reply:
That is most intuitive
"Texting Alias (optional)"
Nobody has voiced any opposition and it's nice and descriptive, especially with the "(optional)" added to it. 

Yay!!!